Streetcode/Streetcode.XUnitTest/MediatRTests/Media/Art/GetByStreetCodeId/GetArtByStreetcodeIdTest.cs

```diff
@@ -90,12 +90,12 @@ private List<Art> GetArtsList()
                 new Art()
                 {
                     Id = 1,
-                    Image = new Image()
+                    Image = new DAL.Entities.Media.Images.Image(),
                 },
                 new Art()
                 {
                     Id = 2,
-                    Image = new Image()
+                    Image = new DAL.Entities.Media.Images.Image(),

```

Comment:
We can use something called Alias Derective:
1 'import' entity in the head of a file:
using ImageEntity = Streetcode.DAL.Entities.Media.Images.Image;
2 use ImageEntity in the code as follows:
Image = new ImageEntity()